tool/pom.xml

```diff
@@ -42,6 +42,11 @@
 			<artifactId>javax.json</artifactId>
 			<version>1.0.4</version>
 		</dependency>
+		<dependency>
+			<groupId>com.ibm.icu</groupId>
```

Comment:
@bhamiltoncx how big is this lib, btw? I make a big jar of all dependencies for deployment.